### ISSUE TITLE
Import error in video display cell of pyttitools-PYTTI_local.ipynb

### ISSUE THREAD

**Issue by ANaka** [pytti-tools/pytti-notebook#33]
Running the cell to embed a rendered video in `pyttitools-PYTTI_local.ipynb` yields an import error for me
```python
# Optionally display your video in the notebook

from Ipython.display import Video

output_video_filename = "default.mp4" # you'll probably need to change this after the first run

Video(output_video_filename)
```

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/home/naka/code/side/pytti-notebook/pyttitools-PYTTI_local.ipynb Cell [1](vscode-notebook-cell:/home/naka/code/side/pytti-notebook/pyttitools-PYTTI_local.ipynb#ch0000015?line=0)6' in <cell line: 3>()
      1[ # Optionally display your video in the notebook
----> ]()[3](vscode-notebook-cell:/home/naka/code/side/pytti-notebook/pyttitools-PYTTI_local.ipynb#ch0000015?line=2)[ from Ipython.display import Video
      ]()[5](vscode-notebook-cell:/home/naka/code/side/pytti-notebook/pyttitools-PYTTI_local.ipynb#ch0000015?line=4)[ output_video_filename = "default.mp4" # you'll probably need to change this after the first run
      ]()[7](vscode-notebook-cell:/home/naka/code/side/pytti-notebook/pyttitools-PYTTI_local.ipynb#ch0000015?line=6)[ Video(output_video_filename)

ModuleNotFoundError: No module named 'Ipython']()
```
Happy to take a stab at it

**Comment by ANaka** [comment]
Ah nevermind, looks like this is fixed in #32

**Comment by dmarx** [comment]
lol, thanks for reminind me to merge
